tools/src/format.rs

```diff
@@ -844,19 +863,18 @@ fn generate_commit_info(tree_name: &str,
     let git = try!(config::get_git(tree_config));
     let hg = match git.hg_map.get(&commit.id()) {
         Some(hg_id) => {
-            let hg_link = format!("<a href=\"{}/rev/{}\">{}</a>", config::get_hg_root(tree_config), hg_id, hg_id);
+            assert!(tree_config.paths.hg_root.is_some());
```

Comment:
I don't think this assert is really helpful given unwrap kinda does the same, but no big deal.

Reply:
Removed